src/os_maild/os_maild_client.c

```diff
@@ -347,7 +347,8 @@ MailMsg *OS_RecvMailQ(file_queue *fileq, struct tm *p, MailConfig *Mail, MailMsg
 MailMsg *OS_RecvMailQ_JSON(file_queue *fileq, MailConfig *Mail, MailMsg **msg_sms)
 {
     int i = 0, sms_set = 0, donotgroup = 0;
-    size_t body_size = OS_MAXSTR - 3, log_size;
+    size_t *body_size = malloc(sizeof(size_t)), log_size;
```

Comment:
Use `os_malloc()` instead of calling `malloc` directly. Same for the others occurrences.

Reply:
Done